README.md

```diff
@@ -303,6 +303,20 @@ default value for `-repotoken`.
 
 You can use the `-v` flag to see verbose output.
 
+
+## Gitlab CI
+
+Store your Coveralls API token as an [Environment Variable](https://docs.gitlab.com/ee/ci/variables/#create-a-custom-variable-in-the-ui).
+
+```
+COVERALLS_TOKEN=your_token_goes_here
+```
+
+```
+$ go get github.com/mattn/goveralls
+$ goveralls -service=gitlab -repotoken=$COVERALLS_TOKEN
```

Comment:
If the environment variable is `COVERALLS_TOKEN` then it is not needed to pass `-repotoken`, which is in fact insecure.

Also, wouldn't it be better to provide here a YAML snippet instead?

Reply:
Better?